src/audits/DataTableHeadersMissing.js

```diff
@@ -0,0 +1,67 @@
+goog.require('axs.AuditRules');
```

Comment:
Missing licence comment at start (copy it from another audit and update the date to 2015)